js/plugins.js

```diff
@@ -280,6 +280,25 @@ plugins.factory('userPlugins', function() {
     imagePlugin.name = 'image';
 
     /*
+     * mp4 video Preview
+     */
+    var videoPlugin = new Plugin(function(message) {
+
+        var url = message.match(urlRegexp);
+        var content = null;
+
+        if (url) {
+            url = url[0]; /* Actually parse one url per message */
+            if (url.match(/\.(mp4|webm|ogv)$/)) {
+                return '<video width="100%"><source src="'+url+'"></source></video>';
```

Comment:
Maybe that's a little bit big. I don't have a strong opinion on this, so what would you prefer? 75% 50%?